ansible/roles/ocp4-workload-rhtr-iot-managed/tasks/provision_fuse.yaml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Get data from samples-registry-credentials in openshift 
+  k8s_facts:
+      api_version: v1
+      kind: Secret
+      name: samples-registry-credentials
+      namespace: openshift
+  register: samples_registry
+
+- set_fact:
+    samples_registry: "{{ samples_registry | replace('.dockerconfigjson','dockerconfigjson') }}"
+
+- set_fact:
+    docker_config_json: "{{ samples_registry.resources[0].data.dockerconfigjson | trim }}"
+    
+- name: Subscribe Operator first, because it takes too long 
+  include_tasks: single_user.yaml
+  with_sequence: start=1 end={{ num_users }} 
+  when:
+    - num_users|int > 0
+
+- name:  pause for 2 minutes, because operator takes sometime to run, it may fail, just re-run  
+  pause:
```

Comment:
Can we change this to a retry loop rather than a pause? Pick what you are looking for and then

```
retries:
delay:
until:
```